Sources/Helpers/OAResourcesUIHelper.mm

```diff
@@ -668,6 +845,11 @@ + (void) startDownloadOfCustomItem:(OACustomResourceItem *)item onTaskCreated:(O
 }
 
 + (void) offerDownloadAndInstallOf:(OARepositoryResourceItem *)item onTaskCreated:(OADownloadTaskCallback)onTaskCreated onTaskResumed:(OADownloadTaskCallback)onTaskResumed
+{
```

Comment:
Do not pass view controllers to methods, instead create a method with a callback that will show the alert.